### DISCUSSION TITLE
Any plans to support XSA?

### DISCUSSION THREAD

**Discussion by ThePlenkov** [SAP/terraform-provider-cloudfoundry#128] [General]
Hi! 

Thanks for picking up this topic. Indeed supporting CF on BTP to manage services, subscriptions, roles and users  - it's a great initiative, however CF is cloud only solution

It would be nice if you could also suppport XSA. It was SAP choice to rename cf to xs, but we all understand that technically it's same technology. So would be nice if SAP also would take care of companies using on-prem solutions built on top of Hana, not only cloud ones.

Thanks!

**Comment by vipinvkmenon** [comment]
Hi @ThePlenkov 

Thanks for your interest and suggestion. 

This CF terraform provider's main focus is to be feature-complete with the Cloudfoundry Foundry Runtime and V3 APIs. Ensuring that we have productive GA release and its stability is our main aim for now.

Having said that, we'll keep this discussion open to revisit it later.
